samples/HttpRequestFeaturesv2/README.md

```diff
@@ -0,0 +1,43 @@
+# About this sample
+This sample extracts HTTP request features and adds them as context features to a Rank request that can be sent to an Azure Personalizer instance. After this request is sent, the user has the option to send a followup Reward request and then regenerate a new Rank request.
+
+This is a [ASP.NET Core MVC with Knockout.js](https://docs.microsoft.com/en-us/aspnet/single-page-application/) solution. When run, it displays a single page with the Rank request, Reward request, and HTTP request features shown. Pressing the **Send Request** button above the Rank request body sends it to your Personalizer instance with HTTP information gathered from the featurizer as part of the context features. Once sent, the **Send Request** button above the Reward request body can be pressed in order to send a corresponding reward. At any time, you can press the **Generate Request** button to create a new Rank request.
```

Comment:
Can we please rename the text on the buttons to match with the previous comment - so "Send Reward Request" and "Send Rank Request". We can simplify the rest of the paragraph too.